script/Ciel/Day 15/15_01B_CIEL15B_1.txt

```diff
@@ -2274,13 +2277,13 @@ All I ever dream of are the events of that night, and the things I got up to bef
 -- 僕の現実、僕が知り得た知識、そこから想像できるであろう未来は、とても狭くて救われないものばかりだから、そういったものを想像する事さえできない」
 My reality,
  my acquired knowledge,
- and the futures I can picture based on that knowledge are all so small and bereft of salvation that I can't even conceive of such a thing."
+ and the futures I can picture based on that knowledge are all so small and without salvation that I can't even conceive of such a future."
```

Comment:
bereft a collocation with salvation, would prefer it

Reply:
doesn't quite match the tone and "without salvation" seems more common anyway

Reply:
Do we want what is common, or what is _good_? Jokes aside, 'without' just doesn't flow nicely here. Prefer bereft, devoid of.